apps/nextjs/next.config.js

```diff
@@ -138,37 +161,39 @@ const getConfig = async (phase) => {
 
 module.exports = getConfig;
 
-module.exports = withSentryConfig(module.exports, {
-  // For all available options, see:
-  // https://github.com/getsentry/sentry-webpack-plugin#options
+if (!process.env.TURBOPACK) {
```

Comment:
Turbopack has trouble with sentry in hot reloading mode apparently, but this doesn't resolve it for me